### PR TITLE
[components] Pass down the Popover context, if any, in Select

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: packages/components/src/components/Select/Select.tsx [hunk 1]
```diff
@@ -63,7 +68,7 @@ export function Select<T extends object>({
           </Button>
           {description && <Text slot="description">{description}</Text>}
           <FieldError>{errorMessage}</FieldError>
-          <Popover>
+          <Popover {...popOverProps}>
```
Comment (stevepiercy) [inline packages/components/src/components/Select/Select.tsx:71]: Is the weird casing deliberate? I think it should be either  `PopoverProps` or `popoverProps` as ["popover" is a single word](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API).

```suggestion
          <Popover {...popoverProps}>
```

**Review by stevepiercy — COMMENTED** [review]
Please check strange casing.

**Comment by sneridagh** [comment]
@stevepiercy thanks!

**Review by stevepiercy — APPROVED** [review]
I approve this commit message. 🤣

**Pull request merged**